### ISSUE TITLE
PHP 8.2: Error "Creation of dynamic property Tqdev\\PhpCrudApi\\OpenApi\\OpenApiBuild "

### ISSUE THREAD

**Issue by micmor** [mevdschee/php-crud-api#944]
```
class OpenApiBuilder
{
    private $openapi;
    private $records;
    private $columns;
    private $status;     // BugFix:  declare  property  $status !!!!
    private $builders;

    public function __construct(ReflectionService $reflection, array $base, array $controllers, array $builders)
    {
        $this->openapi = new OpenApiDefinition($base);
        $this->records = in_array('records', $controllers) ? new OpenApiRecordsBuilder($this->openapi, $reflection) : null;
        $this->columns = in_array('columns', $controllers) ? new OpenApiColumnsBuilder($this->openapi) : null;
        // Error in original code:  $this->status isn't declared
        $this->status = in_array('status', $controllers) ? new OpenApiStatusBuilder($this->openapi) : null;
        $this->builders = array();

```

**Comment by mevdschee** [comment]
Fixed in v2.14.16. Thank you for your contribution.

**Comment by abdulsalamamtech** [comment]
wonderfull
